sources/README.md

```diff
@@ -667,6 +667,20 @@ _Service Centre of the Federal Government for Geo-Information and Geodesy_
 * license: _https://www.govdata.de/dl-de/by-2-0_
 * usage: `concordance`, `property`, `geometry`
 
+## Landesamt für Digitalisierung, Breitband und Vermessung
+
+_Bavarian state office for digitisation, broadband and measurement_ 
+
+* added: `2023-03-17`
+* id: `1830479083`
+* name: `de-by`
+* prefix: `de-by`
+* license_type: _CC BY 4.0_
+* license_text: _You are free to Share, copy and redistribute the material in any medium or format and Adapt, remix, transform, and build upon the material for any purpose, even commercially. _
+* url: _https://www.ldbv.bayern.de/produkte/opendata/opendata_weitere.html_
+* license: _https://creativecommons.org/licenses/by/4.0/deed.de_
+* usage: `geometry`
```

Comment:
Do we want to import any properties and concordances, too? (The answer is probably...)

Reply:
+1 to adding properties and concordances.. there are (at least) names we could import as properties, and an `OBJID` field in some properties that look to be unique concordances.